### DISCUSSION TITLE
View debates

### DISCUSSION THREAD

**Discussion by alexjuiceman** [voteflux/digipol#171] [Ideas]
Would it be possible to include a "view debate/hansard" feature for those bills which have spent some time on the floor already? If that's difficult or out of scope to be in app possibly considering a link to a service that already does this like [Open Australia](https://www.openaustralia.org.au/debates/?id=2020-12-08.85.2)

**Comment by KipCrossing** [comment]
The DigiPol mission statement is *Connecting Real People with their Parliament*.

I would say it definitely in scope; its just not on the horizon.

A larger question is: how do we decide what features to include? At this stage, the features that are included are the ones people are willing to do the work for or to pay for. For example, @BenBallingall wanted a search bar, so he paid me to add/make one.

**Reply by alexjuiceman** [reply]
What's the going rate for feature adding?

Does Digipol have any framework for governance either independently or through any connection with The Flux Party?
